custom/src/main/java/com/splunk/opentelemetry/servicename/WildflyServiceNameDetector.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Splunk Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.splunk.opentelemetry.servicename;
+
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+class WildflyServiceNameDetector extends AppServerServiceNameDetector {
+  private static final Logger log = LoggerFactory.getLogger(WildflyServiceNameDetector.class);
+
+  WildflyServiceNameDetector(ResourceLocator locator) {
+    super(locator, "org.jboss.modules.Main", true);
+  }
+
```

Comment:
wildfly deploys applications from expoded directories if a marker file is present (something like appname.dodeploy). should we try to filter out exploded apps that won't be deployed?

Reply:
I think it's probably fine not to do that right now; just leave a TODO about it, we can come back to it if somebody asks for it.